articles/imagesharp/pixelbuffers.md

```diff
@@ -82,29 +89,16 @@ await Task.Run(() =>
 ```
 
 ### Exporting raw pixel data from an `Image<T>`
-You can use @"SixLabors.ImageSharp.Image`1.TryGetSinglePixelSpan*" to access the whole contigous pixel buffer, for example, to copy the pixel data into an array. For large, multi-megapixel images, however, the data must be accessed and copied per row:
+You can use @"SixLabors.ImageSharp.Image`1.CopyPixelDataTo*" to access the whole contiguous pixel buffer, for example, to copy the pixel data into an array. For large, multi-megapixel images, however, the data must be accessed and copied per row:
```

Comment:
```suggestion
You can use @"SixLabors.ImageSharp.Image`1.CopyPixelDataTo*" to copy the pixel data to a user buffer. Note that the following sample code leads to to significant extra GC allocation in case of large images, which can be avoided by processing the image row-by row instead.
```